k8s/perftest/common/divorce/div-emca.yaml

```diff
@@ -41,6 +41,6 @@ spec:
         enabled: true
     global:
       environment: perftest
-      subscriptionId: "eddbcec4-6ebc-4507-9f22-38d53a92e761"
```

Comment:
This was div-perftest appinsights ID which I assume is wrong, I've change it to the ID of the `DCD-CNP-QA` subscription which is where `perftest` is